src/main/java/org/fcrepo/spec/testsuite/crud/ExternalBinaryContent.java

```diff
@@ -82,6 +198,31 @@ public void postCreateExternalBinaryContentProxy(final String uri) {
                         "request entity, as indicated by a link with " +
                         "rel=\"http://fedora.info/definitions/fcrepo#ExternalContent\"",
                 "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalUri, "proxy")),
+                new Header(SLUG, info.getId()));
+        doPost(uri, headers);
+    }
+
+    /**
+     * 3.9-A-3b PostCreate Proxy File URI
+     *
+     * @param uri of base container of Fedora server
+     */
+    @Test(groups = { "SHOULD" })
+    @Parameters({ "param1" })
+    public void postCreateExternalFileUriBinaryProxy(final String uri) {
+        final TestInfo info = setupTest("3.9-A-3b", "postCreateExternalFileUriBinaryProxy",
+                "Fedora servers should support the creation of LDP-NRs with content external to the " +
+                        "request entity, as indicated by a link with " +
+                        "rel=\"http://fedora.info/definitions/fcrepo#ExternalContent\"",
+                "https://fcrepo.github.io/fcrepo-specification/#external-content", ps);
+
+        final Headers headers = new Headers(
+                new Header(LINK, externalContentHeader(externalFileUri, "proxy")),
+                new Header(SLUG, info.getId()));
```

Comment:
Please avoid using the `Slug` header where possible as support for it is not required AFAIK.